PowerPointLabs/PowerPointLabs/ActionFramework/Action/EffectsLabBlurSelectedActionHandler.cs

```diff
@@ -1,22 +0,0 @@
-﻿using PowerPointLabs.ActionFramework.Common.Attribute;
-using PowerPointLabs.ActionFramework.Common.Extension;
-using PowerPointLabs.ActionFramework.Common.Interface;
-
-namespace PowerPointLabs.ActionFramework.Action
-{
-    [ExportActionRibbonId("EffectsLabBlurSelectedButton")]
-    class EffectsLabBlurSelectedActionHandler : ActionHandler
```

Comment:
I changed Blur Selected to a dynamic menu instead of a button. The dynamic menu is "Blurriness" so that Blur Remainder and Blur All Except Selected can use.